en/docs/install-and-setup/setup/security/user-account-management-for-dev-portal.md

```diff
@@ -0,0 +1,178 @@
+# User Account Management for the Developer Portal
+
+The following sections explain how to manage Developer Portal related user accounts.
+
+## Password recovery
+
+The password recovery feature does not work out-of-the-box because an email server needs to be configured to be able to send the password recovery email. 
+
+Follow the steps below to enable the password recovery feature for the Developer Portal:
+
+1. Shut down the server if it is running.
+
+2. Open the `<API-M_HOME>/repository/conf/deployment.toml` file.
+
+3. Add the following properties to configure the a e-mail server. 
+
+     For more information, see [Configuring the Email Sending Module](https://is.docs.wso2.com/en/5.10.0/setup/configuring-email-sending/).
+
+    ``` java
+    [output_adapter.email]
+    from_address = "wso2am@gmail.com"
+    username = "wso2amtest"
+    password = "Wso2@am70"
+    hostname = "smtp.gmail.com"
+    port = "587"
+    enable_start_tls = true
+    enable_authentication = true
+    ```
+
+4. Re-start the server.
+
+## Account locking
+
+You can lock user accounts with the Account recovery and credential management feature, pre-installed in WSO2 API Manager.
+
+-   [Account locking by failed login attempts](#account-locking-by-failed-login-attempts)
+-   [Account locking by an administrative user](#account-locking-by-an-administrative-user)
+
+### Account locking by failed login attempts
+
+The following steps show how to enable account locking. See [Account Locking by Failed Login Attempts](https://is.docs.wso2.com/en/5.10.0/learn/account-locking-by-failed-login-attempts/) for more information. 
+
+1.  Sign in to the Management Console (`https://<API-M_HOST>:<API-M_PORT>/carbon`) using admin credentials.
+
+2.  Click **Resident** under **Identity Providers** found in the **Main** tab.
+
+3.  Expand the **Login Policies** tab.
+
+4.  Expand the **Account Locking** tab and tick the **Account Lock Enabled** checkbox. Click **Update** to save changes.
+
+    ![enable-account-lock]({{base_path}}/assets/img/administer/product-security/identity-management-for-the-api-dev-portal/account-lock-config.png)
+
+!!! info
+    An error message similar to the following will be logged in wso2carbon.log when the account is locked.
+    
+    ``` java
+    ERROR - Account is locked for user alex in user store PRIMARY in tenant carbon.super. Cannot login until the account is unlocked.
+    ```
+
+
+### Account locking by an administrative user
```

Comment:
same

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1555/commits/dcc2e157314cf300788569c01c2ad89f0877c4b3